src/motion-queue.h

```diff
@@ -72,9 +72,23 @@ struct MotionSegment {
 #endif
 } __attribute__((packed));
 
+//Define a struct to easily access the status register
```

Comment:
Don't write what you do ('Define a struct'), but rather what it is. "Layout of the status register" for instance.